docs/src/markdown/EmptyState.mdx

```diff
@@ -0,0 +1,61 @@
+import Box from '@mui/material/Box';
+import EmptyStateImage from './images/emptyState.png';
+
+# Empty State
+
+The `<EmptyState>` component is an element that can be used as a placeholder when no data is present (such as an empty list, or a placeholder page for future content). This is only used when no data is available, rather than during loading (see [empty states pattern](https://brightlayer-ui.github.io/patterns/empty-states)).
+
+<Box
+    sx={{
+        width: '100%',
+        textAlign: 'center',
+    }}
+>
+    <Box
+        sx={{
+            width: '100%',
+            maxWidth: '200px',
+        }}
+        component="img"
+        alt="Empty State Image"
+        src={EmptyStateImage}
+    />
+</Box>
+
+## Empty State API
+
+<Box>
+
+| Prop Name   | Description                           | Type                | Required | Default |
+| ----------- | ------------------------------------- | ------------------- | -------- | ------- |
+| actions     | Additional components to render below | `ReactNode`         | no       |         |
+| classes     | Style overrides                       | `EmptyStateClasses` | no       |         |
+| icon        | The primary icon                      | `ReactNode`         | yes      |         |
+| description | The secondary text to display         | `ReactNode`         | no       |         |
+| title       | The main text to display              | `ReactNode`         | yes      |         |
+
+</Box>
+
+Any other props supplied will be provided to the root element (`div`).
+
+### Style Overrides
+
+You can override the default styles used by Brightlayer UI by:
+
+-   using the `sx` prop
+-   passing a `classes` prop with keys from the `Name` column below
+-   using the `Global CSS Class` in your main stylesheet
```

Comment:
As per our last PR comments need to remove `` from `Global CSS Class`.
```suggestion
-   using the Global CSS Class in your main stylesheet
```

Reply:
ok did not know this. In the latest stylesheet PR thats merged we have 6 mdx files for drawer that are still using back-quotes here. I'll open issue or story to revisit all the previous api doc files